neoexchange/core/migrations/0064_auto_20220128_2217.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.1.13 on 2022-01-28 22:17
+
+from django.db import migrations
+
+
+def set_stellar_tracking(apps, schema_editor):
+    Block = apps.get_model('core', 'Block')
+    for star_block in Block.objects.exclude(calibsource=None):
+        star_block.tracking_rate = 0
+        star_block.save()
+
+
+class Migration(migrations.Migration):
```

Comment:
Think it's worth putting a comment in this class to say that this is a Data Migration and include a link to the docs (https://docs.djangoproject.com/en/3.1/topics/migrations/#data-migrations)